src/tests/blueprint/t_blueprint_mesh_relay.cpp

```diff
@@ -845,3 +845,45 @@ TEST(conduit_blueprint_mesh_relay, sparse_topos)
 }
 
 
+
+//-----------------------------------------------------------------------------
+TEST(conduit_blueprint_mesh_relay, spiral_multi_file_yaml_json_hdf5)
+{
+    Node io_protos;
+    relay::io::about(io_protos["io"]);
+    bool hdf5_enabled = io_protos["io/protocols/hdf5"].as_string() == "enabled";
+    bool silo_enabled = io_protos["io/protocols/conduit_silo_mesh"].as_string() == "enabled";
+    Node data;
+
+    // use spiral , with 7 domains
+    conduit::blueprint::mesh::examples::spiral(7,data);
+
+    std::vector<std::string> protocols = {"yaml","json"};
+    if(hdf5_enabled)
+    {
+        protocols.push_back("hdf5");
+    }
+
+    // TODO: finish wire up and we can add this case
+    // if(silo_enabled)
+    // {
+    //     protocols.push_back("silo");
+    // }
+    
+    for(auto protocol : protocols)
+    {
+        std::cout << "testing protocol: " << protocol << std::endl;
+        std::string output_base = "tout_relay_spiral_mesh_save_proto_" + protocol;
+        std::string output_dir  = output_base + ".cycle_000000";
+        std::string output_root = output_base + ".cycle_000000.root";
+        relay::io::blueprint::write_mesh(data, output_base, protocol);
+
+        // make sure we can load back, this tests the auto detection
+        // of the file type
+        // read the mesh back in diff to make sure we have the same data
+
+        Node n_read, info;
+        relay::io::blueprint::read_mesh(output_root,n_read);
```

Comment:
I guess you still need to diff?

Reply:
The error before would fail to even open the file, throwing an exception. 
I think we are ok without a diff.

Reply:
then I would remove the comment saying you are going to diff

Reply:
gotcha, cut and paste issue thanks for pointing that out